src/component_timeseries.py

```diff
@@ -0,0 +1,36 @@
+from dash import html
+import dash_bootstrap_components as dbc
+
+
+timeseries-buttons = html.Div(
+            [
+                dbc.RadioItems(
+                    id="timeseries-col",
+                    className="btn-group",
+                    inputClassName="btn-check",
+                    labelClassName="btn btn-outline-primary",
+                    labelCheckedClassName="active",
+                    options=[
+                        {"label": "GDP", "value": 'gdpPercap'},
+                        {"label": "Life Expectancy", "value": 'lifeExp'}
+                    ],
+                    value='gdpPercap',
+                )
+            ],
+            className="radio-group col-md-4 mt-3",
+        )
+
+
+timeseries-card = 
```

Comment:
Convert into single shippable component. Component should be of type list (as expected by `children` arg in dbc.Row)